typings/organizationInvitation.d.ts

```diff
@@ -0,0 +1,14 @@
+import { MetaSys, MetaSysProps, MetaLinkProps } from './meta'
+
+export interface OrganizationInvitationProps {
+  firstName: string,
+  lastName: string,
+  email: string,
+  role: string,
+}
+
+export interface OrganizationInvitation extends MetaSys<MetaSysProps> {
+  organizationMembership: { sys: MetaLinkProps },
+  user: null,
```

Comment:
is `null` correct here?

Reply:
As far as I understand I think so. 
Until user not accept invitation, there is no `user` associated with this invitation and we will always get null, I think.

Reply:
@phoebeschmidt ok, seems like it can be object or null. Updated. Does this make sense? 🧐

Reply:
This is correct.
When you create an invitation it's addressed to an e-mail address. The invitee then has to assign a user to it to be able to accept.
Part of me thinks that this could be omitted from the response but it is in fact part of the invitation object.

Reply:
but is it always null? so when you "getInvitation(id)", after the invitation has been accepted, it will be an object, won't it?

Reply:
You're right @phoebeschmidt. It would be a user object then.